configuration/reports/reportdescriptors/dataexports/sql/all_medication_prescribed.sql

```diff
@@ -0,0 +1,103 @@
+set sql_safe_updates = 0;
+
+drop table if exists temp_medication_orders;
+create table temp_medication_orders
+(
+encounter_id int,
+patient_id int,
+emr_id varchar(20),
+visit_id int,
+order_id int,
+orderer int,
+concept_id int,
+location_id int,
+order_drug text,
+order_formulation text,
+order_location varchar(255),
+order_created_date date,
+order_date_activated date,
+user_entered varchar(255),
+order_quantity int,
+order_quantity_units varchar(50),
+order_quantity_num_refills int,
+order_dose int,
+order_dose_unit varchar(50),
+order_route varchar(50),
+order_frequency_id int,
+order_frequency varchar(50),
+order_duration int,
+order_duration_units varchar(50),
+order_reason text,
+order_comments text,
+product_code varchar(25),
+prescriber varchar(255)
+);
+
+insert into temp_medication_orders (
+encounter_id, 
+patient_id,
+order_id,
+concept_id,
+order_created_date, 
+order_date_activated
+)
+select 
+encounter_id,
+patient_id,
+order_id,
+concept_id,
+date(date_created),
+date(date_activated)
+from orders  where voided = 0;
```

Comment:
I think you need to filter for drug orders here.  Otherwise, you'll get lab, radiology, pathology etc... orders.  (also it will run much faster)